docs/src/architecture/transactions.md

```diff
@@ -57,7 +57,7 @@ As opposed to the previous transaction, this transaction consumes a single note
 Under this model transferring assets between accounts requires two transactions as shown in the diagram below.
 
 <p align="center">
-    <img src="../diagrams/protocol/transaction/Transaction_Flow.png">
+    <img src="../diagrams/architecture/transaction/Transaction_Flow.png">
 </p>
```

Comment:
The content in this section seems somewhat redundant with the content in the "architecture" section - but we can probably fix this in the future.